apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/FlushableConsumerBase.cs

```diff
@@ -150,14 +151,14 @@ private void Flush(IEnumerable<T> items)
         /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         /// <param name="items">The items to flush.</param>
         /// <returns>A task representing the asynchronous operation.</returns>
-        private Task FlushAsync(IEnumerable<T> items, CancellationToken cancellationToken = default)
+        private async Task FlushAsync(IEnumerable<T> items, CancellationToken cancellationToken = default)
         {
             if (items == null || !items.Any())
             {
-                return CommonUtils.CompletedTask;
+                await CommonUtils.CompletedTask.ConfigureAwait(false);
```

Comment:
This is a semantic change - we used to return without having called ReceiveAsync. With this change, you're awaiting a completed task and then calling ReceiveAsync anyway.